### DISCUSSION TITLE
Flutter app

### DISCUSSION THREAD

**Discussion by srrdasa** [iSoron/uhabits#704] [Feature Requests]
Can we have the app built in flutter so it is multi platform compatible

**Comment by iSoron** [comment]
Flutter prototypes and proof-of-concepts are welcome, but I find it unlikely that we will ever migrate our entire project to Dart, especially since we are already halfway towards migrating to Kotlin Multiplatform, which solves the same issues. I am also concerned a number of long-standing issues that Flutter still seems to have in some platforms (see, for example https://github.com/flutter/flutter/issues/61450).
